### ISSUE TITLE
Week19: Stone_fir: 电商系统v1.0: 6

### ISSUE THREAD

**Issue by learnprogram2** [learnprogram2/framework_documents#36]
I hope I can finish this in two weeks.

**Comment by learnprogram2** [comment]
week14-week15: I work some.

**Comment by learnprogram2** [comment]
week 16, I finish some, I will finish the first part in the next two weeks.

**Comment by learnprogram2** [comment]
this week 18, I move my home.
1. Monday and Tuesday, I looking for a new house for rent. finally, I find one more far from the company I work for, but it more comfortable,  clean, and cheaper than before. I love this room best for the sunshine I could enjoy on winter afternoon because the memory of college come back to me, I love this. I love my past life. (weep my tear...)
2. Wednesday, I went to sign the rent contract. 
3. Thursday, I forget what I had done, I think I watch some courseware and videos.
4. Friday, I move my house, very tired. I moved twice, the things I have are so many. I think I should discard some of them! I am a vagrant in this city, in this country, in this society, in this country, and at this time.
5. Saturday, I watch some courseware and videos. and visit my girlfriend at dusk.
6. Today, I go to rescind my old contract for my security deposit.
